### ISSUE TITLE
Add a form to enter an airline and flight number, and verify that the airline name entered is an actual/valid airline name

### ISSUE THREAD

**Issue by airmanopus** [airmanopus/drivinandflyin#162]
Add a space to enter an airline and flight number ~~, and the year/month/day the flight either arrives or departs. (Including the arrival or departure airport is optional.)~~.  This can probably fit into the 'guide' div. Once the data is entered change that div to the carrier, flight number, and codes for departure and arrival airport.

**Comment by airmanopus** [comment]
Looking into autocomplete for at least the airline name field; *for right now*, not going to try to implement it. We can get a list of active airlines on a particular date from https://developer.flightstats.com/api-docs/airlines/v1 to use as a dataset; but (again *for right now*) since we're only working in the US we don't need/want every possible airline-- ideally, just the ones that are flying in/out/within the US today.

**Comment by airmanopus** [comment]
checking airline name (AMERICAN)
matched in first position AMERICAN AIRLINES AMERICAN
matched in first position AMERICAN LINEHAUL CORPORATION AMERICAN

**Comment by airmanopus** [comment]
Next up:
- [ ] Handle the case where we didn't find _any_ possible matches for any of the airline name, the airline ICAO, or the airline IATA. Probably make the input fields red, possibly add an alert.
- [ ] Handle the case where we tried to search by name, but got several airline names back that matched. Possibly change the airline name input field to a populated dropdown???
- [ ] Handle the case where we got a single matched airline; now we try to do a flightStatus() by airline, flight and date. If we get a flight back as a result, we can display flightDetails. If not we need to display an error message (see above).  ALSO: if we get a valid flight back, change the 'flightGuide' div to show the name of the airline and the flight number
